### DISCUSSION TITLE
Scala Support

### DISCUSSION THREAD

**Discussion by tpittmanVS** [renovatebot/renovate#8012] [Suggest an Idea]
What is the current level of support for Scala and associated tools (sbt, mill)? If not already supported, is Scala on the product roadmap for 2021?

**Comment by rarkins** [comment]
We support sbt today: https://docs.renovatebot.com/modules/manager/sbt/

There is no request or plans for mill, so please submit a detailed feature request as an issue if you're interested in it, or even a PR.
